src/Toolkit/Str.php

```diff
@@ -499,9 +442,6 @@ public static function excerpt($string, $chars = 140, $strip = true, $rep = ' 
 	/**
 	 * Convert the value to a float with a decimal
 	 * point, no matter what the locale setting is
-	 *
-	 * @param string|int|float $value
-	 * @return string
 	 */
 	public static function float($value): string
```

Comment:
```suggestion
	public static function float(string|int|float $value): string
```

Reply:
This is true based on the removed docblock but not quite true to reality, e.g. at least `null` has been also supported.

Reply:
Good point, `null` as well. But that should be it then, all other types will break the code I think.